### PR TITLE
Feature/veteran endpoint

### PR DESCRIPTION
**Pull request by MatthewEllisTx**
What work was done?
Made an endpoint /veterns to get all veterans, and wrote tests for it

### CODE REVIEW

File: data/seeds/013-recipients.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const recipients = [
     recipient_middle_name: 'tree',
     recipient_last_name: 'ross',
     recipient_date_of_birth: '1970-10-12',
-    recipient_veteran_status: false,
+    recipient_veteran_status: true,
```
Comment (jaypdl) [inline data/seeds/013-recipients.js:8]: Good catch realizing the seed needed to be updated too!

In the future could implement Faker to create a large amount of seed recipients if DS needs it.

**Review by jaypdl — APPROVED** [review]
Code looks good! Great job writing tests, and also catching that the seed file needed to change!

**Review by samplesn08 — APPROVED** [review]
Great job writing tests to make sure everything run properly!  Very clean code

**Review by DomenicScarcella — APPROVED** [review]
I don't see any obvious snafus.  Good work!

**Pull request merged**